[DOC] Add docs for SORTABLE (#2678)
RediSearch/RediSearch: docs/commands/ft.create.md

    * **SORTABLE**

>         Numeric, tag (not supported with JSON) or text attributes can have the optional SORTABLE argument. As the user [sort the results by the value of this attribute](/redisearch/reference/sorting), they will be available with very low latency. (this adds memory overhead so do not declare it on large text attributes).

Reviewer: ```suggestion
        Numeric, tag (not supported with JSON) or text attributes can have the optional **SORTABLE** argument. As the user [sorts the results by the value of this attribute](/redisearch/reference/sorting), the results will be available with very low latency. (this adds memory overhead so consider not to declare it on large text attributes).
```
